litmus-portal/frontend/okteto.yml

```
@@ -14,7 +14,7 @@ forward:
   - 80:80
 resources:
   limits:
-    cpu: "1"
-    memory: 2Gi
+    cpu: "0.5"
```

500m is a more standard notation for half a cpu.

```suggestion
    cpu: 500m
```